tests/helpers/common/Chocolatey/Test-ChocolateyVersionEqualOrHigherThan.ps1

```diff
@@ -10,7 +10,9 @@
     param(
         [NuGet.Versioning.NuGetVersion]$Version
     )
-    $installedVersion = ((Invoke-Choco list -lo -r).Lines | ConvertFrom-ChocolateyOutput -Command List | Where-Object Name -EQ 'chocolatey').Version
+    if (-not ${script:Chocolatey Installed Version}) {

```

Comment:
Don't use variables with spaces.

Reply:
I wanted to ensure we didn't accidentally encounter some variable naming collision... That being said, we control the testing environment to a large degree, so I'm good with changing the name of this variable. Rain also suggested an enhancement to selectively cache (so we can specify `-Cache` during discovery calls, and it'll cache, but if we use it inside tests it'll be "live" data.

Reply:
I've taken the spaces out.